package.json

```
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "definitely-typed",
-    "version": "0.0.3",
+    "version": "0.0.4",
```

Please revert this change. It’s not related to the @types packages inside the repo.

```suggestion
    "version": "0.0.3",
```